modules/jaeger-removing-instance-cli.adoc

```diff
@@ -0,0 +1,65 @@
+////
+This PROCEDURE module included in the following assemblies:
+- rhbjaeger-installation.adoc
+////
+
+[id="jaeger-removing-cli_{context}"]
+= Removing a Jaeger instance from the CLI
+
+
+. Log in to the {product-title} CLI.
++
+----
+$ oc login
```

Comment:
Should we include a variable here for the link to your instance or explain more about how to use oc login?

`oc login <link to your instance>`